src/vss_tools/vspec/vssexporters/vss2samm/vss2samm.py

```diff
@@ -10,35 +10,17 @@
 # Convert all vspec input files to a ESMF - Aspect Model Editor (SAMM) - ttl formatted file(s).
 #
 
-
-import importlib
 from pathlib import Path
 
 import rich_click as click
 import vss_tools.vspec.cli_options as clo
+import vss_tools.vspec.vssexporters.vss2samm.helpers.ttl_helper as ttl_helper
+import vss_tools.vspec.vssexporters.vss2samm.helpers.vss_helper as vss_helper
 from vss_tools import log
 from vss_tools.vspec.main import get_trees
 from vss_tools.vspec.tree import VSSNode
 
-from .config import config as cfg
-
-VSSConcepts = None
-vss_helper = None
-ttl_helper = None
-
-
-def __setup_environment(output_namespace, vspec_version, split_depth: int) -> None:
```

Comment:
We will need to load these in this particular order. Otherwise, there could be some dependencies problems like no or wrong initialized fields related to th eones from above comments.